### PR TITLE
update broken links in linux and mac install pages fix #83

### LINKED ISSUE #83: Links Broken in "Activating a Validator" Prysm documentation
In the [Activating a Validator](https://docs.prylabs.network/docs/install/mac/activating-a-validator/) portion of the documentation, the following links in "Step 1" are currently broken:

- https://docs.prylabs.network/docs/install/mac/mac
- https://docs.prylabs.network/docs/install/mac/activating-a-validator/docker
- https://docs.prylabs.network/docs/install/mac/activating-a-validator/bazel

### PR DESCRIPTION
**Pull request by slowbart**
fixed broken links. by adding absolute paths - noted previous comment suggesting not to use relative paths. First time using git/github/docker for contribution :-)

### CODE REVIEW

**Comment by wgknowles** [comment]
many of the links in this PR are still broken (mainly /glossaries/terminology ones)
Please use this PR instead: #97

File: docs/reading/eth2.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ This page serves material necessary to catch up with the current state of ETH2 d
 
 ## Ethereum client implementations
 
-* [Geth](https://github.com/ethereum/go-ethereum) \(known also as go-ethereum\) is the Golang implementation of [ETH1](../glossaries/terminology.md#eth1)
+* [Geth](https://github.com/ethereum/go-ethereum) \(known also as go-ethereum\) is the Golang implementation of [ETH1](/docs/glossaries/terminology#eth1)
```
Comment (wgknowles) [inline docs/reading/eth2.md:98]: /docs/terminology/#eth1

Comment (slowbart) [reply docs/reading/eth2.md:98]: yep. I was just flagging a few issues I came across reading the docs. Didn't intend for a comprehensive update :-)  So.. I'm new at this ... should I close this PR or leave it open for other updates?

Comment (wgknowles) [reply docs/reading/eth2.md:98]: I didn't mean to double-up on the work. I should have checked for existing PR's before i started.  Your work is much appreciated! I see you caught some bad anchors that I missed, need to make sure those get in.

**Comment by rauljordan** [comment]
Resolved by #97

**Pull request closed without merging**